realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -244,6 +246,7 @@ public Builder(Context context) {
                 throw new IllegalArgumentException("A non-null Context must be provided");
             }
             initializeBuilder(context.getFilesDir());
+            ReLinker.loadLibrary(context, "realm-jni");
```

Comment:
Doesn't this cause problem with our JNI testing? If the case is only about Table, the `Builder` will not be called. And also, I think put this in a static block would be a good idea since it only needs to be ran once.